lib/logstash/inputs/lumberjack.rb

```diff
@@ -126,4 +127,69 @@ def start_buffer_broker(output_queue)
       end
     end
   end
+
+  class Decoration < LogStash::Inputs::Base
+    public
+    def decorate(event)
+      super(event)
+    end
+  end # class Decoration
```

Comment:
@urso Yeah, Okay for the protected, for me I'll take is as friendly reminder that maybe it should be done elsewhere.